upgrade/1.1.24/01_make_alumni_standalone.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO email_virtual_domains (name) VALUES ("alumni.polytechnique.org");
+UPDATE email_virtual_domains SET aliasing=id WHERE id=LAST_INSERT_ID();
+INSERT INTO email_virtual_domains (name) VALUES ("alumni.m4x.org");
+UPDATE email_virtual_domains SET aliasing=id WHERE id=LAST_INSERT_ID();
```

Comment:
See comment on IRC: aliasing of alumni.m4x.org should be the id of alumni.polytechnique.org inserted just beforehand.